webrender_traits/src/types.rs

```diff
@@ -322,20 +349,32 @@ pub struct GlyphKey {
     pub size: Au,
     pub index: u32,
     pub color: ColorU,
+    pub x_suboffset: SubpixelOffset,
+    pub y_suboffset: SubpixelOffset,
```

Comment:
How come? Skia uses 4 for each. Also, there are vertical text languages that have subpixel text as well.